packages/fluentui/react-bindings/src/styles/getStyles.ts

```diff
@@ -22,18 +22,17 @@ const getStyles = (options: ResolveStylesOptions): GetStylesResult => {
   // - conditionally add sources for evaluating debug information to component
 
   const resolvedVariables = resolveVariables(
-    options.displayName,
+    options.displayNames,
     options.theme,
     options.props.variables,
     options.performance.enableVariablesCaching
   );
-
   const { classes, resolvedStyles, resolvedStylesDebug } = resolveStyles(options, resolvedVariables);
 
   // conditionally add sources for evaluating debug information to component
   if (process.env.NODE_ENV !== 'production' && isDebugEnabled) {
     options.saveDebug({
-      componentName: options.displayName,
+      componentName: options.displayNames[0],
```

Comment:
This should probably be composed or all displayNames, or the last one I guess..

Reply:
I replaced it with `componentName: options.displayNames.join(':'),`. Will it be enough for now?